Library/Formula/txr.rb

```diff
@@ -0,0 +1,16 @@
+class Txr < Formula
+  desc "Original, New Programming Language for Convenient Data Munging"
```

Comment:
Will do! (honestly, I thought it looked weird too, but I was just reproducing what was on the official website)